### PR TITLE
Feature/update nginx component

### PR DESCRIPTION
**Pull request by wibimaster**
Hi, 

There is some nginx keywords that missing ; can you merge this PR please ? :)

Thanks !

### CODE REVIEW

**Comment by wibimaster** [comment]
Done ;)
Sorry for the missing :)

File: tests/languages/nginx/keyword_feature.test [hunk 1]
```diff
@@ -529,6 +536,7 @@ xslt_types
 	["keyword", "min_delete_depth"],
 	["keyword", "modern_browser"],
 	["keyword", "modern_browser_value"],
+        ["keyword", "more_set_headers"],
```
Comment (RunDevelopment) [inline tests/languages/nginx/keyword_feature.test]: @wibimaster Thanks for the fast change! 
But please use tabs for indentation.

**Comment by RunDevelopment** [comment]
@wibimaster 
Perfect!

@mAAdhaTTah I think we are ready to merge.

**Comment by mAAdhaTTah** [comment]
@wibimaster Thanks for the contribution!

@RunDevelopment Thanks for your help!

**Pull request merged**